config/initializers/backtrace_silencers.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
 # Be sure to restart your server when you modify this file.
```

Comment:
I'd expect the blank line to be right after the # frozen... line.

Reply:
I'll move the blank line as you suggest.